crates/net/discv4/src/config.rs

```diff
@@ -121,16 +123,17 @@ impl Default for Discv4Config {
             /// Every outgoing request will eventually lead to an incoming response
             udp_ingress_message_buffer: 1024,
             max_find_node_failures: 5,
-            ping_interval: Duration::from_secs(300),
+            ping_interval: Duration::from_secs(60 * 10),
```

Comment:
why'd this double? any impact

Reply:
the enforced bond duration is an hour, this should be sufficient to cycle through all of them in an hour